src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -153,30 +176,29 @@ def generate_MCP_1_default_config(self, filename):
                     'certFile': 'server_b.cert',
                     'keyFile': 'server_b.key',
                     'certificateHash': 'will be provided in future',
-                    'cbsdRecord': [sas_th_1_cbsd_iter_0, sas_th_1_cbsd_iter_1, sas_th_1_cbsd_iter_2],
+                    'sasThIterationData': [sas_th_1_iteration_0_data,sas_th_1_iteration_1_data],
                     'fileServePath': 'sas_harness_1/'
                 }
         # Create the actual config.
         iter1_config = {
-            'cbsdRecords': cbsd_records_iter_1,
+            'cbsdRequests': [cbsd_records_iter_1_domain_proxy_0,cbsd_records_iter_1_domain_proxy_1],
             'protectedEntities': protected_entities_iter_1,
-            'dpaActiveList': [dpa_1, dpa_2],
-            'dpaDeactiveList': [],
-            'deltaIap': 2
+            'dpaActivationList': [dpa_1, dpa_2],
```

Comment:
FYI (no action needed at this time) we may need to load additional information on the DPAs which are used in this test case. I'll make sure you're on the email thread for this topic so that you have the latest information.

Reply:
Thanks. 

Informational   Comment, no response required